jwql/tests/test_msata_monitor.py

```diff
@@ -305,13 +305,14 @@ def test_plt_mags_time():
 
 
 @pytest.mark.skipif(ON_GITHUB_ACTIONS, reason='Requires access to central storage.')
-def test_mk_plt_layout():
+def test_mk_plt_layout(tmp_path):
```

Comment:
This one was writing to the output path in central storage, but was failing for me because a previous user had written test files there that could not be overwritten.

Reply:
I'll see if I can get Maria to delete the file that is currently in the directory. I think a good way around this is to use the [set_permissions](https://github.com/spacetelescope/jwql/blob/develop/jwql/utils/permissions.py#L25) function on the output file. Then it will always be owned by jwqladm and should have permissions set such that it can be overwritten. That way anyone should be able to run the tests locally.

Reply:
Checking now, I don't see a file there, but the test directory is not group-writeable so the test still fails.  Should the set_permissions go in the monitor code or the test?

Reply:
Maria said she's deleted the file. So hopefully the test will work now.

Reply:
I've asked her to update permissions for the directory. I think set_permissions should go in the test for sure, since that is creating this version of the file. The monitor itself is always run by the administrative accounts on the jwql servers, so in that case I think everything is ok. @mfixstsci do you agree?

Reply:
Okay, thanks.

The same issue exists for the wata_monitor - can you please have her update that file/directory too?

Reply:
I passed that along. @penaguerrero

Reply:
Thanks for the fix, Maria!

I pushed an update that writes the file to the expected directory and sets group write permission on it so it can be overwritten.